src/latexify/codegen/function_codegen_test.py

```diff
@@ -40,6 +40,25 @@ def test_visit_functiondef_use_signature() -> None:
     assert FunctionCodegen(use_signature=True).visit(tree) == latex_with_flag
 
 
+def test_visit_functiondef_ignore_docstring() -> None:
+    tree = ast.FunctionDef(
+        name="f",
+        args=ast.arguments(
+            args=[ast.arg(arg="x")],
+            kwonlyargs=[],
```

Comment:
Thoughts on removing these empty fields from the tests? Would make it easier to read imo / remove some bloat

Reply:
How about using `ast.parse` as in some other tests?

```python
tree = ast.parse(textwrap.dedent("""\
    def f(x):
        '''docstring'''
        return x
    """)
```

Reply:
```python
def test() -> None:
    tree = ast.parse(
        """
def f(x):
    '''docstring'''
    return x"""
    )
```

I've gone with this format. Unfortunately due to the nature of multiline strings it seems like the `def` can't have any preceding indentation, which makes it a bit awkward to read, but still better than before.

There can't be any space between the last non-whitespace character and the closing `"""`.

I decided to opt against the starting `\` because it doesn't actually add anything in terms of the test, and I felt that it's better to keep it simple for readability.

Reply:
> could probably add this function into test_utils if it's commonly used

I think this is not recommended according to the DAMP principle. It is somewhat desirable to write self-contained tests:
https://abseil.io/resources/swe-book/html/ch12.html#tests_and_code_sharing_dampcomma_not_dr